src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculation.kt

```diff
@@ -29,9 +29,9 @@ data class SentenceCalculation(
   var numberOfDaysToLicenceExpiryDate: Long = 0
   var licenceExpiryDate: LocalDate? = null
 
-  //  Home Detention Curfew Expiry Date(HDCED)
-  var numberOfDaysToHomeDetentionCurfewExpiryDate: Long = 0
-  var homeDetentionCurfewExpiryDateDate: LocalDate? = null
+  //  Home Detention Curfew Eligibility Date(HDCED)
+  var numberOfDaysToHomeDetentionCurfewEligibilityDate: Long = 0
+  var homeDetentionCurfewEligibilityDateDate: LocalDate? = null
```

Comment:
DateDate? 😆

Reply:
I just really wanted to make sure people knew it was a date